lib/jazzy/sourcekitten.rb

```diff
@@ -448,6 +458,7 @@ def self.autolink(docs, root_decls)
         doc.abstract = autolink_text(doc.abstract, doc, root_decls)
         doc.return = autolink_text(doc.return, doc, root_decls) if doc.return
         doc.children = autolink(doc.children, root_decls)
+        doc.declaration &&= autolink_text(doc.declaration, doc, root_decls, true)
```

Comment:
If I refactor this to `doc.declaration = .. if doc.declaration` the line gets too long and rubocop complains :( Should I refactor anyway and disable the rubocop warning?